tests/test_utils.py

```diff
@@ -1,35 +1,89 @@
-# import pytest
-# from agoradatatools.etl import utils
-# from synapseclient import Synapse
+import pytest
+from agoradatatools.etl import utils
+from unittest.mock import patch
+import synapseclient
+import yaml
+import io
+import builtins
 
-# file_object = {
-#     "id": "syn25838546",
-#     "format": "table",
-#     "final_filename": "teams",
-#     "provenance": [],
-#     "destination": "syn25871921"
-# }
 
+class TestLoginToSynapse:
```

Comment:
Bruno's code reminded me that this was a feature.  Can you try this:

```
@pytest.mark.usefixtures("syn")
```

Reply:
This does not seem to pass the fixture to the whole class, was that the idea?

Reply:
Ah, I didn't realize that - thanks for testing.  That was not the intention -just thought it looked cleaner than the current solution.